test/deploy.sh

```diff
@@ -81,7 +81,7 @@ function launch_localnet() {
     i=$((i + 1))
 
     # Read config for i-th node form config file
-    IFS=' ' read -r ip port mode bls_key shard <<<"${line}"
+    IFS=' ' read -r ip port mode bls_key shard node_config <<<"${line}"
```

Comment:
What is this change and why is it part of this PR?

Reply:
This one allows to set a custom config file for each single node in local test net. So, we are able to enable/disable any RPC and test the RPC filter. By default it uses default configuration which all RPCs are enabled. Actually this change could be very helpful for other tests as well. 
If the config file is not set in local test net config file, then it will use the default values like how it works so far.
The test net config file is in this path: 
`github.com/harmony-one/harmony/test/configs/local-resharding.txt`
It includes a single line for each single node like this:
`127.0.0.1 9000 validator .hmy/65f55eb3052f9e9f632b2923be594ba77c55543f5c58ee1454b9cfd658d25e06373b0f7d42a19c84768139ea294f6204.key
127.0.0.1 9001 validator .hmy/40379eed79ed82bebfb4310894fd33b6a3f8413a78dc4d43b98d0adc9ef69f3285df05eaab9f2ce5f7227f8cb920e809.key
127.0.0.1 9002 validator .hmy/02c8ff0b88f313717bc3a627d2f8bb172ba3ad3bb9ba3ecb8eed4b7c878653d3d4faf769876c528b73f343967f74a917.key
...
`
So, if we add a config file path to the end of any line, that node will use that custom config file. An example is shown below:
`127.0.0.1 9000 validator .hmy/65f55eb3052f9e9f632b2923be594ba77c55543f5c58ee1454b9cfd658d25e06373b0f7d42a19c84768139ea294f6204.key null ./bin/node1_config.toml`
It makes local test net more flexible and we are able to feed different configs to each single node.